duckduckgo/macos-browser
# Fix an autoconsent exception on old Safari

Task/Issue URL: https://app.asana.com/0/1177771139624306/1205239910258201/f
Tech Design URL:
CC: @kdzwinel @sammacbeth @shakyShane

**Description**:
Due to the updated build pipeline, previous autoconsent version caused exceptions in older Webkit versions (versions of Safari older than 16.4)

**Steps to test this PR**:
1. run the app on the oldest supported macOS/WebView version
2. enable CPM
3. confirm that CPM works (e.g. on https://doodle.com)
4. confirm no CPM exceptions on that and other pages



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
